deployments/ansible/roles/collector/defaults/main.yml

```diff
@@ -1,16 +1,18 @@
 ---
 # defaults file for splunk-otel-collector role vars
-
+# Note: To modify vars based upon ansible_os_family  change the values of vars in tasks/vars.yml
```

Comment:
The comment looks confusing. Why user would have to change values in tasks/vars.yml instead of using the provided set of config options?